tests/grid/filtered_iterator.cc

```diff
@@ -140,9 +140,9 @@ void test ()
       bool (*predicate) (const active_cell_iterator, const unsigned int)
         = &level_equal_to<active_cell_iterator>;
       FilteredIterator<active_cell_iterator>
-      begin (std_cxx11::bind (std::ptr_fun(predicate), std_cxx11::_1, 3),
+      begin (std_cxx11::bind (predicate, std_cxx11::_1, 3),
```

Comment:
This may have become pointless with the change to std::bind calls. My earlier modification avoided changing logic where not explicitly necessary.